.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: Build
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  workflow_dispatch:
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Install .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.0.x
+
+    - name: Build
+      run: dotnet build /p:Configuration=Release
+
+    - name: Upload
+      uses: actions/upload-artifact@v3
+      with:
+        name: LocalizationUtilities
+        path: ./LocalizationUtilities/bin/Release/net6.0/LocalizationUtilities.dll
+        if-no-files-found: error
+        
+    - name: Upload
+      uses: actions/upload-artifact@v3
+      with:
+        name: LocalizationUtilities_nupkg
+        path: ./LocalizationUtilities/bin/Release/ds5678.Modding.TLD.LocalizationUtilities.1.1.0.nupkg
```

Comment:
I think `./LocalizationUtilities/bin/Release/*.nupkg` or `./LocalizationUtilities/bin/Release/**.nupkg` might work here.

If not, change to 2.0.0 similar to the other places.